taxi/views.py

```diff
@@ -64,13 +73,13 @@ class CarDetailView(LoginRequiredMixin, generic.DetailView):
 
 class CarCreateView(LoginRequiredMixin, generic.CreateView):
     model = Car
-    fields = "__all__"
+    form_class = CarCreationForm
```

Comment:
The `CarUpdateView` should use a different form class if the update form differs from the creation form. If the forms are the same, consider renaming `CarCreationForm` to a more generic name like `CarForm` to reflect its dual purpose.